framework/test/mocha/unit/modules/chain/logic/round.js

```diff
@@ -96,6 +96,7 @@ describe('round', () => {
 			round = new Round(scope, task);
 			done();
 		});
+		sinonSandbox.reset();
```

Comment:
this should be in afterEach

Reply:
This test was not atomic so I went for beforEach to avoid having to dig deeper into this test, can we split this to a new issue so the rest of the PR can move forward?